packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -267,7 +272,7 @@ export const CollectionScreenDetailsTile = ({
       renderBottomContent={renderTrackList}
       headerText={!isOfflineModeEnabled ? headerText : undefined}
       renderHeader={isOfflineModeEnabled ? renderHeader : undefined}
-      renderImage={renderCustomImage ?? renderImage}
+      renderImage={renderImage ?? renderImage}
```

Comment:
Idk about this line 😂 

But the custom image render is used for smart collections, like Heavy Rotation and things like that

Reply:
Omg that should be renderImageProo